app/assets/javascripts/leaflet_helper.js

```diff
@@ -165,5 +165,49 @@
          "Cities (zoom in)": city  , 
          "windrose (zoom in)": windrose
     };
+    
+    if(sethash == 1) {
+      var allMapLayers = {
+        "BL1": baselayer,
+
+        "Wisconsin_NM": Wisconsin_NM,
+        "FT_mobile": FracTracker_mobile,
+        "Purple": PurpleAirMarkerLayer,
+        "STruth": SkyTruth,
+        "FracTL": Fractracker,
+        "ToxicR": ToxicRelease,
+        "OdorR": OdorReport,
+        "MapK": MapKnitter,
+        "OIMPower": OpenInfraMap_Power,
+        "OIMapTelecom": OpenInfraMap_Telecom,
+        "OIMPetroleum": OpenInfraMap_Petroleum,
+        "OIMWater": OpenInfraMap_Water,
+        "JMincome": Justicemap_income,
+        "JMamericanIndian": JusticeMap_americanIndian,
+        "JMasian": JusticeMap_asian,
+        "JMblack": JusticeMap_black,
+        "JMmulti": JusticeMap_multi,
+        "JMhispanic": JusticeMap_hispanic,
+        "JMnonWhite": JusticeMap_nonWhite,
+        "JMwhite": JusticeMap_white,
+        "JMplurality": JusticeMap_plurality,
+        "Clouds": clouds,
+        "cloudsclassic": cloudscls,
+        "precipitation": precipitation,
```

Comment:
Oh, CodeClimate is pointing out that we cant have duplicate keys in an assoc. array -- for "precipitation" and "rain" -- maybe rename the second of each? What is "cls"?

Reply:
Done 👍 !